src/h3lib/lib/h3Index.c

```diff
@@ -277,8 +277,9 @@ H3Index H3_EXPORT(h3ToCenterChild)(H3Index h, int childRes) {
  * contiguous regions exist in the set at all and no compression possible)
  * @return an error code on bad input data
  */
+// todo: update internal implementation for int64_t
```

Comment:
Does this block the PR?

Reply:
I don't think it should block. All existing downstream code should still work. Technically, very large examples with >=2^32 cells would probably not work, but I don't think we really have any applications like that. (?)

But also, I'm planning on re-writing `compact` anyway in my next big PR; I've got a few optimizations in mind. That re-write will fix this issue.